src/utils/fetchRockets.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
```

Comment:
Contributors: Francis Wayungi [status: ] & Nimplay Acosta Nuñez [status: ]

- [ ] But I can only see one contributor's name! To make your project approved you both need to comment and participate as well!

Reply:
@Rustamxon7 yes my friend we start with 2 coders but microverse add a thrid parner and the same day he leave the group let me fix the project

Reply:
@Rustamxon7 
the thing is that my partner made a separate branch to upload his changes for that reason only his changes appeared in this pull